runhouse/servers/cluster_servlet.py

```diff
@@ -47,6 +59,8 @@ async def __init__(
         self.cluster_config: Optional[Dict[str, Any]] = (
             cluster_config if cluster_config else {}
         )
+        self.cluster_config["has_cuda"] = detect_cuda_version_or_cpu() != "cpu"
```

Comment:
why do we need to add to the cluster config?

Reply:
1. So we could easily get it from the clusterServlet config (we need in order to decide wether to run the gpu collection thread)
2. It is useful when we print status info of gpu [PRs](https://github.com/run-house/runhouse/pull/1209/files) (we execute `detect_cuda_version_or_cpu()` only once, set it in the ClusterServlet's cluster_config and that use it in different places)

Don't have strong opinion saving it in the cluster_config instead of executing `detect_cuda_version_or_cpu()` each time.

Reply:
ok cool, yea i think theres enough benefits to having it in the config